src/sql-parser/tests/testdata/scalar

```diff
@@ -474,52 +474,52 @@ List([List([Op { op: Op { namespace: [], op: "+" }, expr1: Value(Number("1")), e
 parse-scalar
 LIST[1,2,3][1]
 ----
-SubscriptScalar { expr: List([Value(Number("1")), Value(Number("2")), Value(Number("3"))]), indexes: [Value(Number("1"))] }
```

Comment:
How did this pass tests on the first pr?

Reply:
idk and haven't had time yet to figure it out yet. going to look at it tomorrow or Friday